addons/website_slides/static/src/xml/slide_quiz_create.xml

```diff
@@ -34,39 +36,49 @@
 
     <t t-name="slide.quiz.answer.line">
         <div class="o_wslides_js_quiz_answer row align-items-center mb-1" t-attf-data-answer-id="#{answer ? answer.id : ''}" >
-            <a class="o_wslides_js_quiz_is_correct col-1 text-center" title="Select one good answer">
-                <label class="my-0">
-                    <input t-if="answer and answer.is_correct" class="d-none" type="radio" name="radio" checked="true" />
-                    <input t-else="" class="d-none" type="radio" name="radio" />
-                    <i class="fa fa-lg fa-check-circle-o" />
-                </label>
-            </a>
-            <input type="text" class="form-control col-9" placeholder="Enter your answer" t-attf-value="#{answer ? answer.text_value : ''}"/>
-            <i class="o_wslides_js_quiz_add_answer fa fa-lg fa-plus-circle p-2" title="Add another answer" />
-            <i class="o_wslides_js_quiz_remove_answer fa fa-lg fa-trash-o p-2" title="Remove this answer" />
```

Comment:
Instead, we will:

Add "p-0" on the element `<a class="o_wslides_js_quiz_is_correct col-1 text-center" title="Select one good answer">`
And change the other `<i>` elements just like you did.

That will make only 3 lines of diff and keep everything minimal.